doc/reference/network_bridge.md

```diff
@@ -6,39 +6,50 @@ discourse: 7322
 # Bridge network
 
 As one of the possible network configuration types under LXD, LXD supports creating and managing network bridges.
+<!-- Include start bridge intro -->
+A network bridge creates a virtual L2 ethernet switch that instance NICs can connect to, making it possible for them to communicate with each other and the host.
 LXD bridges can leverage underlying native Linux bridges and Open vSwitch.
+<!-- Include end bridge intro -->
 
-Creation and management of LXD bridges is performed via the `lxc network` command.
-A bridge created by LXD is by default "managed" which means that LXD also will additionally set up a local `dnsmasq`
-DHCP server and if desired also perform NAT for the bridge (this is the default.)
+The `bridge` network type allows to create an L2 bridge that connects the instances that use it together into a single network L2 segment.
+A bridge created by LXD is by default "managed", which means that in addition to the bridge, LXD sets up a local `dnsmasq` DHCP server.
```

Comment:
The use of "by default" here suggests a LXD bridge can be something other than "managed", which it cannot.

Reply:
How about instead:

> LXD created bridges are managed, which means that in addition to the creating the bridge interface itself, LXD also sets up a local `dnsmasq` process to provide DHCP, IPv6 route announcements and DNS services to the network.